File: 日常实验/实验1/task1.py
import cv2  # 导入opencv库
lena_color = cv2.imread("./images/lena.jpg")  # 读取图像lena.jpg
lena_gray = cv2.cvtColor(lena_color, cv2.COLOR_BGR2GRAY)  # 将原图进行灰度化处理
# 通过窗口展示图片
cv2.imshow('lena_color', lena_color)
cv2.imshow('lena_gray', lena_gray)
cv2.waitKey(0)  # 设置waitKey(0),代表按任意键继续

File: 日常实验/实验4/task1.m
lena = imread('lena.jpg');
lena_gray = rgb2gray(lena);

subplot(1, 3, 1);
imshow(lena_gray);
title('原始图像');

PSF = fspecial('motion', 15, 45);
h1 = fspecial('disk', 40);
J1 = imfilter(lena_gray, PSF, 'circular');
J2 = imfilter(lena_gray, h1, 'circular');

subplot(1, 3, 2);
imshow(J1);
title('模糊运动图像');

subplot(1, 3, 3);
imshow(J2);
title('disk模糊图像');
